<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KreuzstichGUI::ImportPictureDialog</class>
 <widget class="QDialog" name="KreuzstichGUI::ImportPictureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import picture</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Original size:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="originalSize_lbl">
       <property name="text">
        <string>(originalSize)</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="width_sb">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="height_sb">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="keepAspectRatio_cb">
       <property name="text">
        <string>Keep aspect ratio</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="maxColors_sb">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>65536</number>
         </property>
         <property name="value">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>colors</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Thread list:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="threadlist_cmb"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="reduceTo_cb">
       <property name="text">
        <string>Reduce to</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>KreuzstichGUI::ImportPictureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>KreuzstichGUI::ImportPictureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>width_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>KreuzstichGUI::ImportPictureDialog</receiver>
   <slot>updateHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>height_sb</sender>
   <signal>valueChanged(int)</signal>
   <receiver>KreuzstichGUI::ImportPictureDialog</receiver>
   <slot>updateWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keepAspectRatio_cb</sender>
   <signal>stateChanged(int)</signal>
   <receiver>KreuzstichGUI::ImportPictureDialog</receiver>
   <slot>updateHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reduceTo_cb</sender>
   <signal>stateChanged(int)</signal>
   <receiver>KreuzstichGUI::ImportPictureDialog</receiver>
   <slot>reduceToChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>127</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateHeight()</slot>
  <slot>updateWidth()</slot>
  <slot>reduceToChanged()</slot>
 </slots>
</ui>
